src/filters/match-filter.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Construct a match filter
+ *
+ * @param  {String}  field    Field name to match over.
+ * @param  {String}  term     Query value.
+ * @param  {Boolean} isPhrase Should the term be treated as a phrase or not.
```

Comment:
Possibly a more flexible approach is to allow an `options` hash as the third argument here, so all of the optional parameters that the elasticsearch match query supports can be supported here. i.e. this could become:

```
@param {Object} options Options
@param {String} options.type Match query type, default to "boolean".
@param {String} options.fuzziness Fuzziness setting.
...
```
See https://www.elastic.co/guide/en/elasticsearch/reference/2.3/query-dsl-match-query.html#query-dsl-match-query-boolean.

Then we don't need a PR to add support for every option elasticsearch provides.